### ISSUE TITLE
commit 6f7dfccda4cc38bd325cb1b59cb5912947c6568d breaks many packages

### ISSUE THREAD

**Issue by zickgraf** [gap-packages/AutoDoc#189]
After commit 6f7dfccda4cc38bd325cb1b59cb5912947c6568d, `make -B doc` in `CAP_project/CAP` leads to an error:

```
XML Parse Error: Line 536 Character 93
Original file: /home/xxxxxx/.gap/pkg/CAP_project/CAP/doc/_Chapter_Category_of_Categories.xml, line number 141.
-----------
where <Math>D_i = A_i</Math> if <Math>b_i = \mathtt<A>false</A></Math>, and <Math>D_i = A_i^<A>\mathrm<A>op</A></A></Math> otherwise.
                                                                                            ^
-----------
Wrong element in A: A
!!! Type 'Show();' to watch the input string in pager - starting with
    line containing error !!!
Or 'ShowOrigin();' to look it up in its source file.
Error,  at /home/xxxxxx/.gap/pkg/GAPDoc/lib/XMLParser.gi:146 called from
ParseError( str, r.start, Concatenation( "Wrong element in ", name, ": ", r.content[Position( l, false )].name ) ); at /home/xxxxxx/.gap/pkg/GAPDoc/lib/GAPDoc.gi:99 called from
CheckAndCleanGapDocTree( x, str ) at /home/xxxxxx/.gap/pkg/GAPDoc/lib/GAPDoc.gi:157 called from
func( elm ) at /usr/lib/gap/lib/coll.gi:1552 called from
ForAll( r.content, function ( x )
      return CheckAndCleanGapDocTree( x, str );
  end ) at /home/xxxxxx/.gap/pkg/GAPDoc/lib/GAPDoc.gi:157 called from
CheckAndCleanGapDocTree( x, str ) at /home/xxxxxx/.gap/pkg/GAPDoc/lib/GAPDoc.gi:157 called from
...  at makedoc.g:20
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
```

This also happens in the following packages:
* https://github.com/homalg-project/FinGSetsForCAP/
* https://github.com/homalg-project/Bialgebroids/
* https://github.com/homalg-project/FunctorCategories/

Is this an intended backwards-incompatible change?

**Comment by sebasguts** [comment]
@zickgraf Thank you for reporting!
@fingolfin I chose `{}` for arguments, which was super duper bad apparently.
We can eiter chose `§` for the arguments, or make the markdown not scan anything else inside math mode. Or just remove the argument scanning again. Which would you prefer.

**Comment by alex-konovalov** [comment]
Ouch. Is this in the just released version, which I haven't yet picked up? Will you be able to make a new release urgently?

**Comment by sebasguts** [comment]
@alex-konovalov You can expect the next release today

**Comment by oysteins** [comment]
I support the idea of using `§` for arguments.

**Comment by sebasguts** [comment]
@oysteins Yeah, that would exactly be what we discussed in Siegen, right? I was just thinking it might look too weird. But better weird that wrong.

**Comment by sebasguts** [comment]
Should be fixed by #190 . Hope nobody of you regularly uses `§` in his math formulas ;)

**Comment by oysteins** [comment]
I don't remember if we discussed exactly that syntax, but it seems reasonable.  After all, there are not that many characters left that are not used for anything else.  For me, the main point is to get a simple way to mark up arguments without cluttering the descriptions with lots of XML tags.
